CONTRIBUTING.adoc

```diff
@@ -30,14 +27,12 @@ https://github.com/openshift/origin/releases[the Releases page].  These are
 Linux, Windows, or Mac OS X 64bit binaries (note that Mac and Windows are
 client only). You'll need Docker installed on your local system
 (see https://docs.docker.com/installation/[the installation page] if you've
-never installed Docker before).
+never installed Docker before and you're not on RHEL/CentOS/Fedora).
```

Comment:
Maybe I should just get rid of this sentence -- it's covered below. On Linux in general you should be using the package manager instead of their install docs.